packages/africa/README.md

```diff
@@ -0,0 +1,73 @@
+# Africa
+
+[![License: CC BY-SA 4.0](https://img.shields.io/badge/License-CC%20BY--SA%204.0-blue.svg)](https://creativecommons.org/licenses/by-sa/4.0/)
+
+Contains all the states and territories (mainland and islands) of Africa:
+* United Arab Emirates
+* Angola
+* Burkina Faso
+* Bahrain
+* Burundi
+* Benin
+* Botswana
+* Democratic Republic of Congo
+* Central African Republic
+* Republic of Congo
+* Côte d'Ivoire
+* Cameroon
+* Cape Verde
+* Djibouti
+* Algeria
+* Egypt
+* Western Sahara
+* Eritrea
+* Ethiopia
+* Gabon
+* Ghana
+* Gibraltar
+* Gambia
+* Guinea
+* Glorioso Islands
+* Equatorial Guinea
+* Guinea-Bissau
+* Kenya
+* Liberia
+* Lesotho
+* Libya
+* Morocco
+* Madagascar
+* Mali
+* Mauritania
+* Malawi 
+* Mozambique
+* Namibia
+* Niger
+* Nigeria
+* Rwanda
+* Seychelles
+* Sudan
+* Sierra Leone
+* Senegal
+* Somalia
+* South Sudan
+* Chad
+* Togo
+* Tunisia
+* Uganda
+* South Africa
+* Zambia
+* Zimbabwe
+
+## Installation
+
+### npm
+
+`npm install --save @svg-maps/africa`
+
+### yarn
+
+`yarn add @svg-maps/africa`
+
+
```

Comment:
Could you add the credits section with all the required information (source, license...) please?